src/class-positioned-post-queries.php

```diff
@@ -62,12 +62,14 @@ public function query( array $args ): Post_Query {
 
 		$out = $this->positioned;
 
+		$backfill_ids_count = count( $backfill_ids );
+
 		// Insert dynamic posts into the available slots in the map of positioned posts.
 		do {
 			if ( null === current( $out ) ) {
 				$out[ key( $out ) ] = array_shift( $backfill_ids );
 			}
-		} while ( next( $out ) !== false && count( $backfill_ids ) > 0 );
+		} while ( next( $out ) !== false && $backfill_ids_count > 0 );
```

Comment:
this needs to stay as it was. There is an `array_shift()` which reduces the size of $backfill_ids during the `while`.

Reply:
Good catch! I'll find another way to satisfy `phpcs`.